aten/src/ATen/native/Convolution.cpp

```diff
@@ -618,8 +635,9 @@ at::Tensor _convolution(
     }
 #endif
   } else if (input.device().type() == c10::DeviceType::CPU || input.device().type() == c10::DeviceType::CUDA) {
-    // TH/native only covers CPU/CUDA implementation.
-    if (params.groups == 1) {
+    if (params.use_caffe2_depthwise(input, weight)) {
+      output = depthwise3x3ConvNCHW(input, weight, bias, params.padding, params.stride, params.groups);
```

Comment:
Will be called for CUDA tensors as well. Not sure that we want it.

Reply:
Thanks for pointing that out!  That was definitely not my intention.